### PR TITLE
Lbiiif16

### PR DESCRIPTION
**Pull request by siddharthbhagwan**
LIBIIIF-16. Image clipper stabilized on image change.
Clipper is kept alive for multiple clippings on the same image as well as on changing the image.
https://issues.umd.edu/browse/LIBIIIF-16

### CODE REVIEW

File: js/src/widgets/imageView.js [hunk 1]
```diff
@@ -581,8 +581,8 @@
               document.body.appendChild(a);
               a.click();
               document.body.removeChild(a);
-              _this.osdSelection.disable();
```
Comment (peichman-umd) [inline js/src/widgets/imageView.js:584]: Rather than just commenting these 2 lines out, go ahead and delete them.

Comment (siddharthbhagwan) [reply js/src/widgets/imageView.js:584]: Done.

**Comment by peichman-umd** [comment]
@siddharthbhagwan One last thing, in your most recent commit, a `.DS_Store` file snuck in. Please remove it. Then this PR will be good to go!

**Comment by siddharthbhagwan** [comment]
Sorry about that. Fixed it.

**Pull request merged**